### ISSUE TITLE
iotagent service was broken

### ISSUE THREAD

**Issue by betefaber** [dojot/dojot#310]
![image](https://user-images.githubusercontent.com/37310063/37485206-9aa259a2-2869-11e8-889d-8295028e5a8f.png)

event that broke the service:
**./emulator.sh --run --config-file=./config-semaforo.json --debug**

```
ubuntu@iotmid03:~/dojot/teste/mqtt-emulator$ cat config-semaforo.json
{
"protocol": {
  "mqtt":{
  "serverAddress": "10.50.11.44",
  "port": 1883,
  "topic": "/onibus"
  }
},
"device":{
  "frequency": 500,
  "duration": 8500,
  "accelerate": 1,
  "sensors": [
      {"name": "localizacao", "value": "FILE(bus-tracking.csv:1)"}

  ]
}
}
ubuntu@iotmid03:~/dojot/teste/mqtt-emulator$ cat bus-tracking.csv
"-22.8861, -47.0780"
"-22.8845, -47.0771"
"-22.8826, -47.0760"
"-22.8801, -47.0753"
"-22.8783, -47.0748"
"-22.8791, -47.0741"
"-22.8807, -47.0729"
"-22.8836, -47.0711"
"-22.8854, -47.0692"
"-22.8875, -47.0668"
"-22.8899, -47.0643"
"-22.8921, -47.0628"
"-22.8938, -47.0610"
"-22.8953, -47.0613"
"-22.8972, -47.0621"
ubuntu@iotmid03:~/dojot/teste/mqtt-emulator$
```

**Comment by mmagr** [comment]
If a MQTT connection is setup with unknown user, clientid and topic schemes, the agent would not produce a valid promise response to the parser, effectively throwing an unhandled exception that would break the service. 

This should be fixed by dojot/iotagent-mosca#3.

**Comment by mmagr** [comment]
This should be fixed on [0.2.0-nightly20180319](https://github.com/dojot-nightly/docker-compose/releases/tag/0.2.0-nightly20180319).

@betefaber can you please verify?

**Comment by mmagr** [comment]
This should be fixed on [0.2.0-nightly20180319](https://github.com/dojot-nightly/docker-compose/releases/tag/0.2.0-nightly20180319).

@betefaber can you please verify?
